ulakbus/services/common/document_render.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from zato.server.service import Service
+
+
+class RenderDocuments(Service):
```

Comment:
please add HAS_CHANNEL attribute which is used by ulakbus manage.py while deploying services. see here: https://github.com/zetaops/ulakbus/blob/develop/ulakbus/services/ogrenci/sinav_plani.py#L16